### DISCUSSION TITLE
Create/save your own template

### DISCUSSION THREAD

**Discussion by DennisvdPluijm** [microsoft/vsts-extension-retrospectives#156] [Ideas]
See this suggestion by @LauritsQueueit: creating your own template (https://github.com/microsoft/vsts-extension-retrospectives/issues/130). I thought it would be good to put here as well.

Would be a great follow-up to the current available templates, if you use a (slightly) different approach for your retrospectives. This can save you from having to changing the column (names), colors and icons every sprint.
A copy function of your current retrospective board could achieve the same result.

**Comment by BobSears53** [comment]
I agree . Having he ability to create and reuse your own template would be a great new feature.

**Comment by GingerBr** [comment]
I agree, this is desperately needed instead of having to recreate the columns for each Retro.

**Comment by gspanjers** [comment]
Yes please, we need templates, or at least be able to change the default. Every time I have to create our board to our specifications.

**Comment by KPurton** [comment]
Yes, we constantly change the template to meet our needs this is time consuming for our team members. We would like to create our own template but cannot.

**Comment by ColinMaring** [comment]
Saving would be nice, but if this takes more time to build, beeing able to create an (empty) copy could be a nice MVP to this feature.

**Comment by timurkaripov** [comment]
Same here, we are constantly adding new column on each Retro

**Comment by tomzosh** [comment]
I would love to create my own columns and save them as my own template as well.  This would greatly enhance the value of this plug in.

**Comment by jasondduncan** [comment]
This would be so beneficial, but I see that it has been asked for over the last 2 1/2 years.  Does anyone still look at this anymore?

**Comment by JStuve** [comment]
> Saving would be nice, but if this takes more time to build, beeing able to create an (empty) copy could be a nice MVP to this feature.

[Open PR](https://github.com/microsoft/vsts-extension-retrospectives/pull/561) that does exactly that. Hoping to get more activity on the extensions.

**Comment by JStuve** [comment]
The ability to create a copy of an existing boards is now release with version 1.92.1! 

I believe this satisfied a lot of requests and will be closing issues and discussion related to board templates/copying boards.
